products/sle/main.pm

```diff
@@ -922,6 +922,26 @@ sub load_feature_tests() {
     loadtest "feature/feature_console/suseconnect.pm";
 }
 
+sub load_online_migration_tests() {
+    # stop packagekit service and more
+    loadtest "online_migration/sle12_online_migration/online_migration_setup.pm";
+    loadtest "online_migration/sle12_online_migration/register_system.pm";
```

Comment:
If use source images which are already registered and the test fail during online migration, then next time to redo online migration with the source images would be broken. because scc side kept registration status of source system which is different with the original registration status of source image, if we did not perform reset registration after failure.
And another reason to register source system is to select addons and modules more flexible.